src/nginx.go

```diff
@@ -72,8 +93,14 @@ func parseStatusURL(statusURL string) (hostname, port string, err error) {
 		return
 	}
 
+	isHTTP := u.Scheme == httpProtocol || u.Scheme == httpsProtocol
```

Comment:
I would avoid having this variable. If you want to have some semantic knowledge then create a function which gives this context.